train_unet.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import json
+import os
+
+import torch
+import numpy as np
+from torch import optim
+from torch.nn import functional as F
+from torch.utils.data import Subset
+from torchvision import transforms as trfs
+from brats import transformations
+from brats.data.datasets import read_dataset_json
+from brats.losses import DiceLoss
+from brats.metrics import DiceScore, metric_for_class
+from brats.models import UNet3D
+from brats.data import datasets
+from brats.training.runners import run_training_epoch, run_validation_epoch
+from brats.training.stop_conditions import EarlyStopping
+from brats.training.loggers import TensorboardLogger, BestModelLogger, BestStateDictLogger, ModelLogger, \
+    StateDictsLogger, log_parameters
+
+if __name__ == '__main__':
+
+    def create_parser():
+        parser = argparse.ArgumentParser(description='Train UNet 3D.')
+        parser.add_argument('--dataset_json', type=str, required=True)
+        parser.add_argument('--division_json', type=str, required=True)
+        parser.add_argument('--log_dir', type=str, required=True)
+        parser.add_argument('--device', type=str, required=True)
+        parser.add_argument('--epochs', type=int, required=True)
+        parser.add_argument('--batch_size', type=int)
+        parser.add_argument('--patience', type=int, default=10)
+        parser.add_argument('--learning_rate', type=float, default=0.001)
+        parser.add_argument('--train_valid_ratio', type=float, default=0.8)
+        parser.add_argument('--input_size', type=int, default=240)
+        parser.add_argument('--progress_bar', type=int, default=0)
```

Comment:
`train_val_ratio` and `progress_bar`  are unused

Reply:
Fixed